client/src/components/clusters/TopCharts.js

```diff
@@ -0,0 +1 @@
+will put code in later, just making the folder
```

Comment:
Comment this out.

You can see that it's currently creating a lint error, since it's not valid javascript: https://github.com/googleinterns/play-web-step-2020/pull/27/checks?check_run_id=808655475